### DISCUSSION TITLE
Configuring Camera Module 3

### DISCUSSION THREAD

**Discussion by Reksaku** [svpcom/wfb-ng#346] [Questions]
Hi, my student association is building a drone to take high-resolution photos of geological sites, and we've chosen Camera Module 3 as our main camera. The main issue we're facing is configuring it with the latest release image. Has anyone successfully tried something similar?

**Comment by svpcom** [comment]
Do you have any sample gstreamer pipeline from the camera manufacturer?

**Reply by Reksaku** [reply]
All I found was this GStreamer pipeline: `gst-launch-1.0 libcamerasrc ! capsfilter caps=video/x-raw,width=1280,height=720,format=NV12 ! v4l2convert ! v4l2h264enc extra-controls="controls,repeat_sequence_header=1" ! 'video/x-h264,level=(string)4.1' ! h264parse ! rtph264pay ! udpsink host=<ip-addr> port=<port>`.  It's from [Raspberry Pi Documentation](https://www.raspberrypi.com/documentation/computers/camera_software.html#gstreamer).
An additional problem is that I am not able to even take a test photo. Using libcamera ends up with the error: "No camera available." I switched off legacy camera support, but nothing changed. 
I suspect it may be related to the wrong version because the libcamera version on the release image is v0.0.1+54-d528119f. Furthermore, I conducted some tests on a clean RPi OS Debian Bookworm, where libcamera version is v0.2.0+46-075b54d5. The camera works fine there.

**Reply by svpcom** [reply]
Try yo use latest RPI image and setup wfb-ng on it using Setup HOWTO. I'll update reference RPI image on the next release
